test-requirements.txt

```diff
@@ -1,5 +1,6 @@
 coverage
 funcsigs
+lxml==4.3.3
```

Comment:
Do you want this in `requirements.txt` as well, to be used on site?

Reply:
Nope - the import checking handles this being missing for production systems and lets you know that validation wont be used if it is missing.